src/hosting/api.ts

```diff
@@ -275,3 +276,64 @@ export async function createRelease(site: string, channel: string, version: stri
   );
   return res.body;
 }
+
+/**
+ * Adds channel domain to Firebase Auth list.
+ * @param project the project ID.
+ * @param url the url of the channel.
+ */
+export async function addAuthDomain(project: string, url: string): Promise<Domains> {
```

Comment:
It's a little hard to keep track of the return values here when one is Domains and the other is string[] - could this and getApprovedDomains be unified to the same return type, maybe `string[]`?